README.md

```diff
@@ -1 +1,47 @@
-# java-web-servlet-filter
\ No newline at end of file
+# OpenTracing Java Web Servlet Filter Instrumentation
+
+[![Travis](https://travis-ci.org/opentracing-contrib/java-web-servlet-filter.svg?branch=master)](https://travis-ci.org/opentracing-contrib/java-web-servlet-filter)
+
+This library provides instrumentation for Java Web Servlet applications.
+
+## Maven
+```xml
+   <dependency>
+     <groupId>io.opentracing.contrib</groupId>
+     <artifactId>opentracing-web-servlet-filter</artifactId>
+     <version>0.0.1.SNAPSHOT</version>
```

Comment:
these are hard to keep up-to-date manually... maybe remove the version from the README? Or link to the sonatype or bintray search for this artifact to reveal the current released version?